libraries/ui-library/src/components/six-input/six-input.tsx

```diff
@@ -48,13 +48,13 @@ export class SixInput {
   labelId = `input-label-${id}`;
   helpTextId = `input-help-text-${id}`;
   errorTextId = `input-error-text-${id}`;
-  input: HTMLInputElement;
+  nativeInput?: HTMLInputElement;
   customErrorText = '';
   customValidation = false;
 
   readonly eventListeners = new EventListeners();
 
-  @Element() host: HTMLSixInputElement;
+  @Element() host?: HTMLSixInputElement;
```

Comment:
As documented in https://stenciljs.com/docs/host-element#element-decorator, it is ok to define this as `@Element() host!: HTMLSixInputElement;`